README.md

```diff
@@ -125,6 +125,7 @@ You can customize some of the view effects programatically or using xml styleabl
 * Draggable view margin right applied when the view is minimized.
 * Draggable view margin bottom applied when the view is minimized.
 * Enable or disable the horizontal alpha effect applied while the view is being horizontally dragged.
+* [NEW] Enable or disable touch on minimized/maximized view to minimize/maximize.
```

Comment:
Please, remove the "[NEW]" and keep the rest of the sentence. I'm going to add a CHANGELOG.md file with this information.